### ISSUE TITLE
ACT-R37 - Black text on white background are flagged as Element has contrast ratio lower than minimum.

### ISSUE THREAD

**Issue by geirsf** [qualweb/act-rules#19]
**Testrule:** ACT-R37
**Tested with** : @qualweb/core": "0.7.32"  and online tool http://qualweb.di.fc.ul.pt/evaluator/https%253A%252F%252Fskattetaten.no
**Webpage**: https://www.skatteetaten.no/person/

### Issue
ACT-R37 produces failed result "Element has contrast ratio lower than minimum" on black text on white background. 

### How to reproduces

1. Run ACT-R37 in  qualweb/core or http://qualweb.di.fc.ul.pt/evaluator/ on page https://www.skatteetaten.no/person/

Just let me know if we can assist in any way resolving this issue. 

### Observation 
**1. Element has contrast ratio lower than minimum.**

Element:
`<div class="nav-heading">Skatt</div>`

Element selector:
 `html > body:nth-child(2) > header:nth-child(3) > div:nth-child(5) > nav:nth-child(3) > div:nth-child(1) > ul:nth-child(1) > li:nth-child(1) > ul:nth-child(2) > li:nth-child(2) > a:nth-child(1) > div:nth-child(1)`

**2. Element has contrast ratio lower than minimum.**
Element:
`<div class="nav-heading">Avgifter</div>`

Element selector:
 `html > body:nth-child(2) > header:nth-child(3) > div:nth-child(5) > nav:nth-child(3) > div:nth-child(1) > ul:nth-child(1) > li:nth-child(1) > ul:nth-child(2) > li:nth-child(3) > a:nth-child(1) > div:nth-child(1)`

**3. Element has contrast ratio lower than minimum.**
Element:
`<div class="nav-heading">Folkeregister</div>`

Element selector: 
`html > body:nth-child(2) > header:nth-child(3) > div:nth-child(5) > nav:nth-child(3) > div:nth-child(1) > ul:nth-child(1) > li:nth-child(1) > ul:nth-child(2) > li:nth-child(4) > a:nth-child(1) > div:nth-child(1)`

**4. Element has contrast ratio lower than minimum.**
Element:
`<div class="nav-heading">Utenlandsk</div>`

Element selector: 
`html > body:nth-child(2) > header:nth-child(3) > div:nth-child(5) > nav:nth-child(3) > div:nth-child(1) > ul:nth-child(1) > li:nth-child(1) > ul:nth-child(2) > li:nth-child(5) > a:nth-child(1) > div:nth-child(1)`

### Screenshots 

![image](https://user-images.githubusercontent.com/24475820/223705375-e988be47-b3a2-4719-aedf-500d3fbf1a6b.png)
![image](https://user-images.githubusercontent.com/24475820/223706249-eff7fe38-fc51-4853-8caa-2033165a4506.png)
